recipes/deepchecks/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set name = "deepchecks" %}
+{% set version = "0.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/deepchecks-{{ version }}.tar.gz
+  sha256: ee6049bfba2bb0e863d027756c2eb29ae2b10285ac7ab7eee3cc0d8e0c9e6a58
+
+build:
+  number: 0
+  # noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
```

Comment:
```suggestion
    - python >=3.8
```

Reply:
Thank you. But then the library is no longer available for `python <3.8` (which the PyPI version supports). From a user's perspective, I don't know if this is a good or bad thing. Having said that, since python 3.6 security-support ended in 2021 ((source][#py-end-of-life-source]), 3.7 support will be over (probably) by 2022 end. So, may be it is not such a bad idea. But I have a dilemma in doing this.

Considering what I stated above, what do you suggest?

[![image](https://user-images.githubusercontent.com/10201242/149257541-e2050410-d9dd-4c56-8af5-43542acae4a5.png)][#py-end-of-life-source]

[#py-end-of-life-source]: https://endoflife.date/python

Reply:
Made it noarch